packages/comet-uswds/src/components/modal/modal.tsx

```diff
@@ -60,9 +60,7 @@ export const Modal = ({
           <h2 className="usa-modal__heading" id={`${id}-heading`}>
             {heading}
           </h2>
-          <div className="usa-prose">
-            <p id={`${id}-description`}>{children}</p>
```

Comment:
This embedded paragraph tends to lead to accessibility issues given that paragraph elements do not allow adding certain html elements, so removing and allowing the developer to add anything they need to the children